Library/Formula/apr-util.rb

```diff
@@ -15,6 +16,7 @@ class AprUtil < Formula
 
   depends_on "apr"
   depends_on "openssl"
+  depends_on "berkeley-db"
```

Comment:
Let's make this `:optional` for now and we can consider making it `:recommended` later if requested by more people.